.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## What?
+
+Describe what changes you are making. e.g. `Adding additional eslint configs to catch potential bugs.`
+
+## Why?
+
+Please provide the reason(s) why you are making this change? Please no one-liners like, `We saw some bugs.`; expand upon the reason like, `Added X rule because it caught a bug in Y component. Added this check since this bug is sure to surface in other components.`
```

Comment:
I was trying to continue to the eslint examples... but yeah, probably a good idea to give a different example. I had a hard to coming up with this one too 😅

Reply:
Yeah, my comment goes for both examples.

Reply:
How about these two suggested changes @deini?
```suggestion
Please provide the reason(s) why you are making this change? Please no one-liners like, `It was missing.`; expand upon the reason like, `Adds X prop because it was missing in the Box component. Documentation notes this prop exists, but it actually wasn't implemented.`
```

Reply:
👍